forms/migrations/0004_auto_20210914_1420.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.2.13 on 2021-09-14 11:20
```

Comment:
Why you have two migration files?

Reply:
I added default=False into answer.ready after the tests failed and runned makemigrations.

But yeah. I combine that little fix onto the original migration file and commit it.